src/components/ExpandableCard.tsx

```
@@ -0,0 +1,22 @@
+import { Card, CardContent } from "@/components/ui/card";
+
+const ExpandableCard = ({ conversation }: { conversation: { id: string; question: string; answer: string } }) => {
+  return (
+    <Card className="bg-white border border-gray-300 rounded-xl shadow-md hover:shadow-lg overflow-hidden cursor-pointer transition-all">
+      <CardContent className="p-4">
+        <details className="group">
+          <summary className="text-lg font-medium text-gray-900 cursor-pointer transition-all group-hover:text-gray-600">
+            {conversation.question}
+          </summary>
+          <div className="mt-2 text-sm text-gray-700 leading-relaxed">
+            <span dangerouslySetInnerHTML={{ __html: conversation.answer }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)